sarc/jobs/series.py

```diff
@@ -218,6 +220,140 @@ def compute_job_statistics(job: SlurmJob):
     )
 
 
+DUMMY_STATS = {
+    label: np.nan
+    for label in [
+        "gpu_utilization",
+        "cpu_utilization",
+        "gpu_memory",
+        "gpu_power",
+        "system_memory",
+    ]
+}
+
+
+# pylint: disable=too-many-statements,fixme
+def load_job_series(
+    *,
+    fields: None | list[str] | dict[str, str] = None,
+    clip_time: bool = False,
+    callback: None | Callable = None,
+    **jobs_args,
+) -> pandas.DataFrame:
+    """
+    Query jobs from the database and return them in a DataFrame, including full user info
+    for each job.
```

Comment:
It would be good to document here what fields the dataframe may contain. It's not a good idea to list all attributes of `Job` nor the series, but we could at least point to them so the user can infer what will be contained. In addition, we would need to mention the new fields generated in this function, namely: gpu_allocated, gpu_requested, unclipped_start, unclipped_end.

Actually, I think it would be best to uniformize the gpu and cpu columns. We should also have `cpu_allocated` and `cpu_requested`.

Reply:
Done: https://github.com/mila-iqia/SARC/pull/89/commits/f67329f18d42740ae51d25c3739aac7f214f76e6